---
geometry_msgs/PoseStamped pose
uint8 code
string message
